src/pybind/mgr/cephadm/services/cephadmservice.py

```diff
@@ -241,7 +242,8 @@ def get_keyring_with_caps(self, entity: AuthEntity, caps: List[str]) -> str:
 
     def _inventory_get_addr(self, hostname: str) -> str:
         """Get a host's address with its hostname."""
-        return self.mgr.inventory.get_addr(hostname)
+        addr = self.mgr.inventory.get_addr(hostname)
+        return socket.getfqdn(addr)
```

Comment:
The method name implies an address, and it's description states return an address but this change results in the return of a shortname or fqdn?

I'd prefer to see a new function for get_fqdn and use that to avoid any future confusion.

Reply:
maybe rename `_inventory_get_addr` -> `_inventory_get_fqdn` and update the docstring? 

the function is only used by the services listed in monitoring.py ...

Reply:
Thank you both for your comments :+1: . And I agree, the name should be changed.

@pcuzner would you be fine with a rename of the method or do you prefer a different one? I'm fine with both ways.

Reply:
Well, I'll go ahead with renaming the method and updating the docstring. @pcuzner please let me know if you prefer a separate method instead :slightly_smiling_face: